### DISCUSSION TITLE
Configure backend from frontend

### DISCUSSION THREAD

**Discussion by fdw** [jonaswinkler/paperless-ng#779] [Feature Requests]
It would be comfortable and better UX if I could configure the backend from the frontend (i.e. what I now can do with environment variables).
For one thing, it's easier to change it via a browser than to restart the docker container with a different set of env vars. Additionally, it's much easier to discover possible settings.

**Comment by jonaswinkler** [comment]
Most of the configuration options in the documentation are used at startup and don't affect the application once started. That's true for pretty much anything related to paths and folders, hosting and services (database, redis, tika).

It's possible to do this for all OCR settings, and *some* of the software tweaks, and it sounds really useful to me.
